crates/snforge-scarb-plugin/src/utils.rs

```diff
@@ -1,30 +1,55 @@
 use cairo_lang_macro::{Diagnostic, Severity};
 use indoc::formatdoc;
 
-fn higher_severity(a: Severity, b: Severity) -> Severity {
+pub fn higher_severity(a: Severity, b: Severity) -> Severity {
```

Comment:
Why was this done?

Reply:
This has to be changed because the [macro](https://github.com/foundry-rs/starknet-foundry/pull/2475/files#diff-13330e2784a18e63e1241cc441655b1332d63a2991f843ed73ae0242cfa2edf6R51) uses this function